### PR TITLE
fix issue with lunr & theme name with space

### PR DESCRIPTION
**Pull request by tomap**
Lunr was not at the same version as in package.json
also there was an issue with image names with space in them
cleanup tags
fix broken links
fix typo

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -62,7 +62,10 @@ gulp.task('revreplace', function() {
         const src = img.attr('data-src') || img.attr('data-org');
         if (!src) return;
 
-        const jpgPath = replaceBackSlash(rename(src, {extname: '.jpeg'}));
+        // url encode the image path to handle cases where there is a space in image name
+        const srcEncoded = src.replace(' ', '%20');
```
Comment (YoshinoriN) [inline gulpfile.js]: Not sure, but maybe [encodeURI](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI) is better than this code.

Comment (tomap) [reply gulpfile.js]: Will test asap

Comment (tomap) [reply gulpfile.js]: It worked. Thank you

**Comment by YoshinoriN** [comment]
@tomap 
Btw, IMHO one pull-request, one concern is better.

**Pull request merged**